ui/Screen/Org/MemberRegistration.svelte

```diff
@@ -99,7 +99,7 @@
           {transaction}
           {subject}
           {payer}
-          transactionDeposits={$session.data.transactionDeposits} />
+          registrationFee={$session.data.registrationFee} />
```

Comment:
I thought the registration fee is only for org and user registration 🤔

Reply:
I oversaw this one! Will fix it.

Reply:
Discussion moved to https://github.com/radicle-dev/radicle-upstream/pull/700#discussion_r458916091